### PR TITLE
[TA-400] A task should be assigned to participant and manager only

### PR DESCRIPTION
**Pull request by AtiaZ**
- This RP was created based on https://github.com/exo-addons/task/pull/16

### CODE REVIEW

File: services/src/main/java/org/exoplatform/task/service/impl/UserServiceImpl.java [hunk 1]
```diff
@@ -140,6 +144,101 @@ private User convertToUser(org.exoplatform.social.core.identity.model.Identity i
...
+        membership.setMembershipType(membershipType);
+        membership.setGroupId(groupId);
+
+        ms.add(membership);
+      }
+    }
+
+    if (ms.size() > 0) {
+      try {
+        final ListAccess<org.exoplatform.services.organization.User> orgUsers = orgService.getUserHandler().findUsersByQuery(query, UserStatus.ANY);
```
Comment (plamarque) [inline services/src/main/java/org/exoplatform/task/service/impl/UserServiceImpl.java]: are you sure you want to search in disabled users here ?

Comment (AtiaZ) [inline services/src/main/java/org/exoplatform/task/service/impl/UserServiceImpl.java]: You are right. PR updated to search only on enabled users.

**Comment by boubaker** [comment]
* Participants can be spaces, groups... The groups aren't indexed by ES, so we can't use it for "mentioning" for tasks
* Unit tests have to be added

**Comment by boubaker** [comment]
The Pr seems ok for me but the build fails with one javadoc error.

**Pull request closed without merging**